docs/reference/action_list.rst

```diff
@@ -578,5 +578,32 @@ Checkbox range selection
     You can check / uncheck a range of checkboxes by clicking a first one,
     then a second one with shift + click.
 
+Displaying a non-model field
```

Comment:
When I want to display a non-model field in the list or in the show, I create a getter
```
public function getCustomField() {
    $computed = ...
    
     return $computed;
}
```

Shouldn't we mention the two possibility with the advantage/disadvantage of each ?

Reply:
I've added that, thanks